### ISSUE TITLE
No longer working? 

### ISSUE THREAD

**Issue by happypokemonhunter** [PokeAlert/PokeAlert#203]
I am having lots of problems with the app. When I run a scan nothing shows up. What's scary is that when I switch over to Pokemon Go none of the Pokestop or Gyms are shown at all. Is PokeAlert affecting Pokemon Go somehow? It usually works again when I close down both apps and restart the phone. Occasionally it will pick up a few Pokemon scans, but it quickly stops working after that. Prior to today I have never had any issues whatsoever. Is anyone having similar issues?? :(

**Comment by arhi3a** [comment]
Me and my friends also have this issue. If you can not see gyms and pokemon it's indicator that you have been IP soft banned so any acc logged with this IP wouldn't recive any informations about gyms, pokestops, pokemons for aprox. 30mins-2h. We can run 2/3 scans mannualy then we have to change IP. It started happening yeasterday around 19:00-20:30 Warsaw Time

**Comment by davidc1223** [comment]
Ive been told its new server side ip detection that softbans IP ranges. Id quit using poke alert immediately

**Comment by deadmanh** [comment]
Working sporadically for me now, maybe use less accounts?

**Comment by davidc1223** [comment]
I think if accounts are greater than 4 on one ip than the ip gets banned. The sporadic working could be explained if your carrier uses dynamic ips. I was able to connect using a different ip by restarting my phone

**Comment by deadmanh** [comment]
Why do you say 4, not 5??. Not trying to give you a hard time, just want to know where you get that number from.

**Comment by davidc1223** [comment]
Conjecture by a friend of mine. It could very reasonably be 5 as well. In other words, theres no proof (that i know of) for 4. I suppose we could test it. Maybe my friend did test already which could be where he got the number from. i just havent asked him.

**Comment by arhi3a** [comment]
My friend is saying that he is scanning 500m with 50acc continously without bans so you could try that.
I scan 2.2 with 230 and i get soft banned after 2/3 scans

**Comment by davidc1223** [comment]
Ok. we're trying to get more information. I forgot that last night I was able to scan my entire town on my home IP for over an hour with no softbans.

One friend of mine was able to scan 5 780m radius areas with ~30 accounts and experienced no softban. His gf has about 20 more accounts and scans more areas and experienced softbans. 

Sorry about that 4 number earlier. lol

**Comment by deadmanh** [comment]
No problem at all, but now we are testing our parameters. I was on vacation in another country and the hotel wifi allowed the guests to play, so I imagine that more than 5, but less then 150 accounts. 

let's keep testing.

**Comment by PokeAlert** [comment]
@happypokemonhunter @arhi3a @davidc1223 @deadmanh @juanjo4x4 
Sorry guys 
This morning I woke with hundred of people asking me why scan were no longer woking. Iquikcly check with my 15 accounts. All good my side.

Turned out all these people were using OVER 50 accounts so my first guess was ... here we go niantic lowered the max account PER ip. 

Later today I realised that the "captcha" was finally being returned by the server. so... that COULD explain the soft ban happening (captcha NOT solved = soft ban or ban?)
